utils/grpc/escrow.go

```diff
@@ -10,6 +11,11 @@ func EscrowClient(addr string) *EscrowClientBuilder {
 	return &EscrowClientBuilder{builder(addr)}
 }
 
+func (b *EscrowClientBuilder) Timeout(to time.Duration) *EscrowClientBuilder {
```

Comment:
You can define `Timeout` on `ClientBuilder`, then you don't need to implement every service.

Reply:
> You can define `Timeout` on `ClientBuilder`, then you don't need to implement every service.

i've define Timeout on ClientBuilder, but it's not working.
i'll try again and add timeout in another pr if needed.